openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1049,17 +1058,29 @@ The Batch Credential Endpoint issues multiple Credentials in one Batch Credentia
+The Client can request issuance of multiple Credentials that can be:
 
-## Batch Credential Request {#batch-credential_request}
+* different Credential Configurations (with possibly different Credential Formats)
+* different Credential Datasets
+* different Credential instances of the same Credential Dataset (with different cryptographic material)
+* a combination of the points above
 
-The Batch Credential Endpoint allows a Client to send multiple Credential Request objects (see (#credential-request)) to request the issuance of multiple Credentials at once. A Batch Credential Request MUST be sent as a JSON object using the `application/json` media type.
+Wallets need to differentiate between these options because they can lead to variations in user experience. While Credentials with the same Credential Configuration but different Credential Dataset should be displayed separately (e.g. vehicle registration credentials for multiple cars), Credentials with the same Credential Configuration and Credential Dataset (e.g. multiple age proofs for unlinkability) should only appear as a single Credential in the Wallet user interface, as these technical details are not relevant to the average End-user. 
 
-The following parameters are used in the Batch Credential Request:
+## Batch Credential Request {#batch-credential-request}
 
-* `credential_requests`: REQUIRED. Array that contains Credential Request objects, as defined in (#credential-request).
+A Client submits a Batch Credential Request to the Batch Credential Endpoint by sending a JSON object containing the following parameters in the entity-body of an HTTP POST request, using the `application/json` media type.
 
-Below is a non-normative example of a Batch Credential Request:
+* `credential_requests`: REQUIRED. Array that contains Batch Credential Request objects, each of these objects may refer to a different Credential Configuration or the same Credential Configuration with a different Credential Dataset.
+  * `format`: REQUIRED if the `credential_identifiers` parameter was not returned from the Token Response. It MUST NOT be used otherwise. See (#credential-request).
+  * `credential_identifier`: REQUIRED if `credential_identifiers` parameter was returned from the Token Response. It MUST NOT be used otherwise. See (#credential-request).
+  * `proofs`: OPTIONAL. Object providing one or more proof of possessions of the cryptographic key material to which the issued Credential instances SHALL be bound to. The `proofs` object contains exactly one parameter named as the proof type in (#proof-types), the value set for this parameter is an array containing parameters as defined by the corresponding proof type. The `proofs` parameter is REQUIRED if the `proof_types_supported` parameter is non-empty and present in the `credential_configurations_supported` parameter of the Issuer metadata for the requested Credential.
```

Comment:
```suggestion
  * `format`: REQUIRED if the `credential_identifiers` parameter was not returned from the Token Response. It MUST NOT be used otherwise. See (#credential-request).
  * `credential_identifier`: REQUIRED if `credential_identifiers` parameter was returned from the Token Response. It MUST NOT be used otherwise. See (#credential-request).
  * `proofs`: OPTIONAL. See (#credential-request).
```

Reply:
actually, this part is a merge conflict with #319. changing a text introduced there to below is probably sufficient (cc @c2bo )
> The individual Credential Request objects MUST NOT contain `credential_response_encryption` and `proof`.